superdesk/media/media_editor.py

```diff
@@ -39,6 +39,10 @@ class MediaEditorResource(Resource):
             'type': 'dict',
             'required': True,
             'mapping': not_analyzed
+        },
+        'renditions': {
+            'type': 'dict',
+            'mapping': not_analyzed
```

Comment:
I'm not sure about mapping here, it's not elastic resource anyhow so it shouldn't be used

Reply:
I'm also not sure, why it was done? Should I remove it for all?

Reply:
I think so

Reply:
Ok.